polygeist/tools/cgeist/Lib/Attributes.cc

```diff
@@ -21,21 +21,21 @@ using namespace mlir;
 
 namespace mlirclang {
 
-static constexpr StringLiteral PassThroughAttrName = "passthrough";
+constexpr llvm::StringLiteral AttributeList::PassThroughAttrName;
```

Comment:
Nit: Is the `llvm::` qualifier necessary here?

Reply:
Done